src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -0,0 +1,392 @@
+
+NUbots workshops may be run live, or you can work through them yourself in your own time.
+
+The majority of resources on this page are free. Some LinkedIn Learning videos are linked here - UoN students can get access to LinkedIn Learning for free.
+
+## General
+
+<details>
+<summary>Git</summary>
+
+The NUbots team uses Git and GitHub in its development workflow. Almost any project done on the NUbots team involves using Git and GitHub.
+
+**Reading:**
+
+- [NUbook: Git and PR Workflow](/guides/general/git)
+
+**Tutorials:**
+
+- [NUbots Workshop: Git and the PR workflow](https://github.com/NUbots/GitDemos)
+
+</details>
+
+<details>
+<summary> C++ </summary>
+
+The C++ programming language is used in the main robot codebase. Modern C++ and advanced C++ concepts are used in the codebase.
+
+**Reading:**
+
+- [C++ Official Documentation](https://en.cppreference.com/)
+
+**Videos:**
+
+- [LinkedIn Learning: C++ Basics](https://www.linkedin.com/learning/c-plus-plus-essential-training)
+- [LinkedIn Learning: C++ Advanced Topics](https://www.linkedin.com/learning/c-plus-plus-advanced-topics?u=75842122)
+- [YouTube: CppCon](https://www.youtube.com/user/CppCon)
+
+**Tutorials:**
+
+- [NUbots Workshop: Modern C++](https://github.com/NUbots/NULearning/blob/master/cpp/moderncpp.md)
+
+</details>
+
+<details>
+<summary>NUClear</summary>
+
+The NUbots robot code uses the NUClear framework. Anyone developing robot code needs to learn how to work with NUClear. NUClear was created at NUbots as part of a Software Engineering FYP.
+
+**Reading:**
+
+- [NUbook: NUClear](/system/foundations/nuclear)
+
+**Videos:**
+
+- [YouTube: NUClear Basic Concepts](https://www.youtube.com/watch?v=Bx4yU-eA6bk&t=17m00s)
+- [YouTube: NUClear Example](https://www.youtube.com/watch?v=Bx4yU-eA6bk&t=32m00s)
+
+**Resources:**
+
+- [GitHub: NUClear](https://github.com/Fastcode/NUClear)
+- [GitHub: NUClear Roles](https://github.com/Fastcode/NUClearRoles)
+- [GitHub: NUClearNet](https://github.com/Fastcode/NUClearNet.js)
+- [GitHub: NUClearExample](https://github.com/Fastcode/NUClearExample)
+
+</details>
+
+<details>
+<summary>Robot Code</summary>
+
+**Reading:**
+
+- [NUbook: Getting Started](/guides/main/getting-started)
+- [NUbook: System Overview](/system/foundations/overview)
+- [NUbook: Troubleshooting](/guides/general/troubleshooting)
+- [NUbook: Glossary](/guides/general/glossary)
+
+**Resources:**
+
+- [GitHub: Robot Code](https://github.com/NUbots/NUbots)
+
+</details>
+
+<details>
+<summary> RoboCup Competition </summary>
+
+**Reading:**
+
+- [NUbook: RoboCup Resources](/team/robocup/resources)
+
+**Videos:**
+
+- [YouTube: Referee Training Video](https://www.youtube.com/watch?v=HHOtAklrcJU)
+- [YouTube: RoboCupSoccer Overview](https://www.youtube.com/watch?v=BN7t9dTbRyI)
+
+</details>
+
+<details>
+<summary> Using Webots </summary>
```

Comment:
(Super minor thing!) you have spaces around some of the `summary` text parts, but not others. It's inconsistent
```suggestion
<summary>Using Webots</summary>
```